WeakAuras/GenericTrigger.lua

```diff
@@ -1009,17 +1039,28 @@ function LoadEvent(id, triggernum, data)
     end
   end
   if data.unit_events then
+    local includePets = data.trigger.use_includePets == true and data.trigger.includePets or nil
```

Comment:
There are two issues here, both a bit subtle but:

1) Since  you only converted the Health trigger, someone could configure a Health trigger for "PetsOnly", and then switch to the Alternate Power trigger. The Alternate Power trigger doesn't have a includePets setting, but the PetsOnly option is still active.

Now, I think the "Alternate Power" trigger might not need the pets only setting.

2) This code is also run for custom triggers, but custom triggers don't have a data.trigger.use_includePets setting. The way this code works is that in GenericTrigger.Add the various settings are extracted and put into a general format. E.g. unit_events are parsed there too. And then LoadEvent + LoadDisplay work on that general format and don't have to care about which trigger it was.

I've fixed both issues in a commit, which I now need to test.

Reply:
I was going to add the option to more generic trigger, was just waiting for review on the health one.